lib/Parse/ParseDecl.cpp

```diff
@@ -770,31 +770,71 @@ Parser::parseDifferentiableAttribute(SourceLoc atLoc, SourceLoc loc) {
   ParserStatus Status;
```

Comment:
This is unused now.

Reply:
Thanks! Done in https://github.com/apple/swift/pull/17428/commits/913f70d6879a70e55305f90f5d5d2fdf4f593d5e.